content/bn/api-gateway.md

```diff
@@ -0,0 +1,32 @@
+---
+title: API গেটওয়ে
```

Comment:
Kindly update the title using the title in English. For example, API গেটওয়ে(API Gateway)